### ISSUE TITLE
Error in py_call_impl(callable, dots$args, dots$keywords) :    TypeError: use() got an unexpected keyword argument 'warn'

### ISSUE THREAD

**Issue by jxf-hub** [rstudio/reticulate#804]
When I used Python in Rstudio, the following error popped up after I typed repl_Python. I tried a lot of methods and found that when I uninstalled numpy package, I could enter it, but it could not be used after entering it.

**Comment by kevinushey** [comment]
Do you see this with the development version of `reticulate`? You can try installing it with:

```
renv::install("rstudio/reticulate")
```

**Comment by jxf-hub** [comment]
I've tried your method, but it still doesn't work out. I think it has to do with numpy. The first image shows that I did not install Numpy. With the Reticulate ::repl_python command I opened the Python interface, but when Importing other installed library files, I found that Numpy was not installed. Figure 2 shows that when I installed Numpy, I re-used the Reticulate :: Repl_Python command and no more access to the Python interface.
![1](https://user-images.githubusercontent.com/56212597/88448473-faf15380-ce70-11ea-9ab8-e86728e2eb39.jpg)
![2](https://user-images.githubusercontent.com/56212597/88448477-fd53ad80-ce70-11ea-98b8-291d11a7ee17.jpg)
